documentation/src/main/DocumentationTemplate.md

```diff
@@ -162,4 +162,4 @@
 
 #include "wiki/FAQ.md"
 
-#include "wiki/AetSuiteMigration.md"
```

Comment:
Why `AetSuiteMigration.md` is deleted from here?

Reply:
The related page described migration from AET version 2.0 to version 2.1. Very old document that is no longer needed by anyone :)

@Skejven , could we also delete the `AetSuiteMigration.md` file from codebase?